Userland/Libraries/LibJS/Runtime/WeakRefPrototype.cpp

```diff
@@ -30,7 +30,7 @@ JS_DEFINE_NATIVE_FUNCTION(WeakRefPrototype::deref)
     auto* weak_ref = TRY(typed_this_object(global_object));
 
     weak_ref->update_execution_generation();
-    return weak_ref->value() ?: js_undefined();
+    return weak_ref->value().visit([](Empty) -> Value { return js_undefined(); }, [](Object* object) -> Value { return object; }, [](Symbol* symbol) -> Value { return symbol; });
```

Comment:
yikes
maybe split this into a few lines?

Reply:
That looks much nicer, thanks!